x-pack/metricbeat/module/azure/_meta/docs.asciidoc

```diff
@@ -10,6 +10,8 @@ Additional azure API calls will be executed in order to retrieve information reg
 
 The azure module mericsets are `monitor`,  `compute_vm` and `compute_vm_scaleset`
 
+CAUTION: Extra Azure charges on metric queries will be generated by this module.
```

Comment:
Costs are mentioned below in this file, might worth adding the `CAUTION` wording, would like to keep there instead.

Reply:
Thanks @narph! What do you think about keep the `CAUTION` wording here to be consistent with AWS and GCP. But also add a link from this `CAUTION` to the section below?
<img width="536" alt="Screen Shot 2020-07-20 at 10 38 43 AM" src="https://user-images.githubusercontent.com/14081635/87962937-39ec6700-ca75-11ea-9d63-141b4a1311f4.png">

I also added the link for both AWS and GCP since there are separate sections to talk about API usage.

Reply:
I think the issue I have with the wording is that we are saying extra charges will be generated by this module, which sounds like a certainty, but that is not the case, there is a high limit of free calls so users in most cases will not be charged. That might scare users away knowing they should expect a cost with it.
Maybe something like: `Additional Azure charges on metric queries may be generated by this module...`.
@dedemorton what do you think?

Reply:
Good point @narph! I will make the change for azure. Thank you!

Reply:
I agree. In fact, that's exactly how I read the statement (to mean that charges are likely).

In this case, I would use IMPORTANT and not CAUTION.

Reply:
Thank you! I changed it to IMPORTANT and also changed the wording for azure.